src/DotNetCore.CAP/Internal/SnowflakeId.Default.cs

```diff
@@ -71,30 +67,15 @@ public SnowflakeId(long workerId)
     /// middle 10 bit: workerId
     /// lowest 53 bit: all 0
     /// </summary>
-    private long _workerId { get; }
-
-    public static SnowflakeId Default()
-    {
-        if (_snowflakeId != null) return _snowflakeId;
-
-        lock (SLock)
-        {
-            if (_snowflakeId != null) return _snowflakeId;
-
-            if (!long.TryParse(Environment.GetEnvironmentVariable("CAP_WORKERID"), out var workerId))
-                workerId = Util.GenerateWorkerId(MaxWorkerId);
```

Comment:
Cannot be removed, placed into the constructor as the default behavior

Reply:
Added a parameterless constructor that utilizes the above method.